implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/DynatraceConfig.java

```diff
@@ -76,8 +85,37 @@ default DynatraceApiVersion apiVersion() {
                 .orElse(DynatraceApiVersion.V1);
     }
 
+    default String metricKeyPrefix() {
+        // returns an empty string if nothing is set in the properties.
+        return getString(this, "metricKeyPrefix").orElse("");
```

Comment:
I could imagine that using `null` spares some code paths but don't recall the util implementation in that detail.

Reply:
i think they are checked almost at the same time in something similar to `isNullOrEmpty`.